.github/actions/phpunit/action.yml

```diff
@@ -0,0 +1,60 @@
+name: 'Run PHPUnit and Behat tests'
+
+description: 'Set up PHP and its dependencies, install WordPress test library along plugins and themes then run PHPUnit and Behat tests.'
+
+inputs:
+    php-version:
```

Comment:
You should take another look at indentation in the 3 yalm files. Sometimes lines are indented with 4-spaces steps instead of 2, and `dependency-versions` uses 7.

Reply:
Done in b7ce019a.